front-end/src/components/LoanDetails/LoanDetails.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useContext, useEffect } from 'react'
+import { Content, Container, Spinner } from 'native-base'
+import { connect } from 'react-redux'
+import { NavigationContext } from 'react-navigation'
+import PropTypes from 'prop-types'
+
+import Detail from './Detail'
+import Chart from './Chart'
+import History from './History'
+
+import { fetchDetail } from '../../lib/actions'
+
+function LoanDetails({ fetchData, loanList, loading }) {
+  const [loan, setLoan] = useState({})
+  const navigation = useContext(NavigationContext)
+  const { id } = navigation.getParam('shortLoanData')
```

Comment:
@Jeffzholy if you put the `id` into redux store instead, you'll be able to get the current loan in each component without the need of passing down